### ISSUE TITLE
Readability

### ISSUE THREAD

**Issue by FallingSnow** [biji/simplenetspeed#8]
There is already a fork fixing the font size on higher dpi monitors, but I feel that font size shouldn't be hard coded.

- Remove font size so that it matches other extensions.
- Separate arrow, speed, and unit. This way the arrow, speed, and unit aren't bouncing left and right as the speed and unit change. The speed and unit can be set to a fixed width off of the font size using `em`.

Let me know what you think.

**Comment by paulle** [comment]
I would support the suggestions of @FalllingSnow.

**Comment by BoBeR182** [comment]
Is it possible to make this user configurable? I personally preferred the way it was before change, as now it takes up roughly 50% more screen real estate for my setup.

**Comment by bianjp** [comment]
I agree with @BoBeR182, the font is too large now, even larger than window titles.
![screenshot from 2017-06-28 10-57-37](https://user-images.githubusercontent.com/4749277/27618748-ad99f060-5bf0-11e7-841b-4d9c3c029227.png)

**Comment by biji** [comment]
The width is not changed, only the font size is not set, it should follow theme font :)

**Comment by paulle** [comment]
Yes, the font is now following the theme font. This is fine, because the look of the upper panel is now more consistent. Thank you for this.
Is it possible to change the width too, in a way adapting to the font size? Bigger font size = greater width?

**Comment by FallingSnow** [comment]
@bianjp The font size follows the theme default. Fixed font sizes should not be addressed at the extension level. In order to fix your issue the width of the container should be based on ems. Also if you want the text to be smaller I suggest altering your theme's font size.

**Comment by j0k3r** [comment]
This is really unreadable.
![image](https://user-images.githubusercontent.com/62333/27822712-e359d778-60a6-11e7-91ae-c2d96ffcf5fa.png)

I've tried to change font size from the gnome settings panel but couldn't find which font should I change to change the one in the top bar.

Can we have a step by step guide?

**Comment by ser** [comment]
It looks even worse on my monitor, I can read only first number. I'm not able to make a screen shot - because when I take it with ctrl-prntscr it does not include the panel itself. 

Maybe you could produce an option to change a font a font size?

**Comment by ser** [comment]
Hi, I it's much better now, I can lower the size with middle click, but still font is too big too fit. Few more sizes down please as options <3

**Comment by conspacer** [comment]
Perfect smaller size on 4k monitor after middle-clicking. Full string fits now. Thanks

**Comment by j0k3r** [comment]
Works fine for me too now.

**Comment by ser** [comment]
No, it does not fit. 

![nono](https://user-images.githubusercontent.com/231537/29360058-5bac451e-82ab-11e7-8574-9b35ee122dbd.png)

**Comment by conspacer** [comment]
@ser have you middle clicked directly on it? There are 2 font sizes available that toggle on mouse middle click (I am not sure how that works on a touchpad if on a laptop..)

**Comment by ser** [comment]
Yup, with a Ph.D in computer science i am able to middle click. It's just I want to have a larger top bar, and a larger top bar comes out with larger characters. I am not a javascript master, though, but I could imagine I could able to set an extension's topbar width - if it is not possible, it's pretty sad. It's possible with a CSS3 I'm familiar with.

**Comment by conspacer** [comment]
I am not familiar with setting the width of the extensions display. We will have to wait for the author (or a fork) to address this. For me the smaller font works perfectly, but of course every screen environment is different. 

![taskbar-smallerfont-netspeed](https://user-images.githubusercontent.com/23648459/29370787-bc630f1e-82a6-11e7-94fa-db306af40ad6.png)

**Comment by FallingSnow** [comment]
@conspacer Can you check to see if https://github.com/biji/simplenetspeed/pull/10 fixes the issue for you?

**Comment by julianctni** [comment]
Hello everybody - just installed the extension. I set the font by middle-click to the same font size the time has (compared it via GIMP). But still the values are getting cut and extended with dots. Isn't it possible to extend the whole width of the extension so that it doesn't get cut? I do not know anything about Gnome Extensions development but in fact it should be easy because it is just CSS?

**Comment by julianctni** [comment]
> Hello everybody - just installed the extension. I set the font by middle-click to the same font size the time has (compared it via GIMP). But still the values are getting cut and extended with dots. Isn't it possible to extend the whole width of the extension so that it doesn't get cut? I do not know anything about Gnome Extensions development but in fact it should be easy because it is just CSS?

I just fixed it by manually adjusting the css of the extension. I just removed the "width" style attributes and now it automatically adjusts to the width it needs :-) furthermore i moved it to the left of the right area so that it won't make other icons move

**Comment by zezao** [comment]
> > Hello everybody - just installed the extension. I set the font by middle-click to the same font size the time has (compared it via GIMP). But still the values are getting cut and extended with dots. Isn't it possible to extend the whole width of the extension so that it doesn't get cut? I do not know anything about Gnome Extensions development but in fact it should be easy because it is just CSS?
> 
> I just fixed it by manually adjusting the css of the extension. I just removed the "width" style attributes and now it automatically adjusts to the width it needs :-) furthermore i moved it to the left of the right area so that it won't make other icons move

Can you help me with that? Did you remove all the 'width' attributes?
